### ISSUE TITLE
`rethinkdb.issues` might better be called `rethinkdb.current_issues`

### ISSUE THREAD

**Issue by larkost** [rethinkdb/rethinkdb#3514]
In working with the `rethinkdb.issues` table I have found myself automatically thinking that I should be finding old (resolved) issues in the table. Intellectually I know that this is a psudo-table, and so is not really a log-like table, but the tendency continues.

I am taking this as a sign that users will be similarly confused by this, and I think that changing the psudo-table's name to `rethinkdb.current_issues` would avoid this confusion.

**Comment by danielmewes** [comment]
I like the name `current_issues`, except that it's longer to type.

@coffeemug Do you have an opinion on this?

**Comment by deontologician** [comment]
+1 same here

**Comment by danielmewes** [comment]
Haven't heard back from @coffeemug yet, but I think we should rename it to `current_issues`.

I realize this requires another chunk of last-minute find&replace updates to the web UI and documentation, but I think it's worth it.
(also mentioning @chipotle)

@Tryneus can you take care of the rename?

**Comment by Tryneus** [comment]
The renaming is up in review 2471.

**Comment by coffeemug** [comment]
Hey guys, sorry I didn't respond sooner. :+1: on renaming

Also, whenever there is consensus in the engineering team, no need to wait for me! If someone says "hey X is sort of confusing, I think Y would be better" and then a bunch of other people say "yep, Y would totally be better", I probably wouldn't be able to add something valuable.

TL;DR: I trust you -- trust yourselves!

**Comment by Tryneus** [comment]
This has been approved and merged to `next` in commit f5df122.  Will be in release 1.16.
